test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo_test_helper
```

Comment:
Please rebase the branch to resolve conflicts. Another module has already added the same dependency, so you don't need to include this file. It is now present in V18:
https://github.com/OCA/sale-workflow/blob/18.0/test-requirements.txt

Reply:
reabase is done.